index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="mob-navbar">
+          <a href="#">Edith</a>
+          <img src="assets/download.png" alt="option" />
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="title-section">
+        <div class="landing-text">
+          <h1>Edith Oga</h1>
+          <h2>Software developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
```

Comment:
- [ ] [OPTIONAL] This is an optional review as it is not part of this project requirement but it would be great if this part of your website looks as close as possible to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177016204-348ead28-8116-4f89-a290-9d955a4d5ded.png)  |  ![image](https://user-images.githubusercontent.com/74814780/177016229-d8bfed20-8b77-4e3a-be1b-02d599e23d94.png)